utility/src/main/java/org/oppia/android/util/system/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+
+kt_android_library(
+    name = "oppia_date_time_formatter",
+    srcs = ["OppiaDateTimeFormatter.kt"],
+    visibility = ["//visibility:public"],
+    deps = [
+        artifact("com.google.firebase:firebase-crashlytics"),
```

Comment:
For the imports in both targets

```kotlin
import javax.inject.Inject
import javax.inject.Singleton
```
this deps dependency works. But I'm wondering if there's a more explicit maven dependency we can use or if this case is ad-hoc enough that it's fine, @BenHenning? I don't see any other usages of @Inject in the repo.

At least I think this is the underlying original dependency, so we could leave it for now.